client/a8c-for-agencies/components/hoc/with-onboarding-tour/hooks/use-onboarding-tour-sections.tsx

```diff
@@ -235,7 +237,7 @@ export default function useOnboardingTourSections() {
 						),
 					],
 					hint: translate(
-						'{{b}}Limited time offer:{{/b}} Migrate your sites to Pressable or WordPress.com and earn up to $10,000*',
+						'{{b}}Limited time offer:{{/b}} Migrate your sites to Pressable or WordPress.com and earn up to $10,000',
```

Comment:
I apologize for missing this earlier.

We should format the currency for accurate translation. 

`import { formatCurrency } from '@automattic/number-formatters';`

Could we also update this at all the places in this component? I see two other places.

Reply:
Good suggestion. this has been fixed 👍